README.md

```diff
@@ -8,49 +8,54 @@ Please find project details at [Wiki](https://github.com/Samsung/TizenRT/wiki) e
 
 ## Contents
 
-> [Environment Setup](#environment-setup)  
-> [How to Build](#how-to-build)  
-> [Supported Board / Emulator](#supported-board--emulator)  
+- [Environment Setup](#environment-setup)
+- [How to Build](#how-to-build)
+- [Supported Board / Emulator](#supported-board--emulator)
 
 ## Environment Setup
 
-TizenRT provides the easiest way to build with the use of [Docker](https://www.docker.com/).  
```

Comment:
Two spaces in markdown mean line change. Do you remove them because it is unnecesary?

Reply:
In this case, it reads better when the markdown is removed, hence the change.